shared/uavobjectdefinition/shareddefs.xml

```diff
@@ -27,6 +27,7 @@
         <option>Acrodyne</option>
         <option>LQG</option>
         <option>LQGLeveling</option>
+        <option>FlipOverRev</option>
```

Comment:
Rev? Revision? Revert? Reverse? Revise?

Reply:
It has to fit in a relatively narrow field places so I was going for not being the maximum width.  Do you have a preferred form?